src/entities/user.entity.ts

```diff
@@ -25,10 +25,8 @@ export class UserEntity extends BaseEntity {
   @OneToOne(() => PartnerAccessEntity, (partnerAccess) => partnerAccess.user, {
     eager: true,
   })
-  partnerAccess?: PartnerAccessEntity;
+  partnerAccess: PartnerAccessEntity;
```

Comment:
was this `?` intended to be removed? if thats the syntax for optional it should stay, if not please teach me what it means 😂